src/AdminConsole/Pages/App/Playground/Client.cshtml

```diff
@@ -27,6 +27,11 @@
             <div class="mt-8">
                 <div class="mt-6">
                     <form v-on:submit.prevent="loginSubmit" href="#" name="signin" method="post">
+                        <div class="hidden sm:block">
```

Comment:
Maybe show a dropdown on mobile so as to offer something beyond `hidden`?

Reply:
Also, since the purposes are dynamic and can theoretically be a very long list, maybe showing a drop down on desktop is also desirable.